### ISSUE TITLE
Gutter styling

### ISSUE THREAD

**Issue by punkJD** [bertrandg/angular-split#128]
I need to have my gutter composed with two elements, so I can show a different cursor and a different highlight when I hover the center of the gutter and when I hover around the center. Is there a way to achieve this the way the gutter is implemented?
Or do you plan to add this in the near future?

**Comment by bertrandg** [comment]
Hi @punkJD, 
Currently it's not possible but i will add a div inside `<div class="as-split-gutter"></div>` to allow specific cases like yours in next `2.0.0-beta.4` which is gonna be publish in coming days.

**Comment by bertrandg** [comment]
Ok cool, did you do that by forking/copying the lib or with another method?

**Comment by punkJD** [comment]
No, i didn't need it. I extended your splitComponent, and in my component instead of using the standard gutter i use my gutter component, and then instead of using your split component i use mine. I only had to copy the gutter directive code.

**Comment by bertrandg** [comment]
Well done! :+1:
